pkg/apiserver/slowquery/queries.go

```diff
@@ -219,10 +219,28 @@ func QuerySlowLogList(db *gorm.DB, req *GetListRequest) ([]SlowQuery, error) {
 		tx = tx.Where("DB IN (?)", req.DB)
 	}
 
+	// more robust
+	if req.OrderBy == "" {
+		req.OrderBy = "timestamp"
+	}
+
+	order, err := getProjectionsByFields(req.OrderBy)
+	if err != nil {
+		return nil, err
+	}
+	// to handle the special case: timestamp
+	// if req.OrderBy is "timestamp", then the order is "(unix_timestamp(Time) + 0E0) AS timestamp"
+	if strings.Contains(order[0], " AS ") {
+		order[0] = req.OrderBy
+	}
+	if order[0] == "timestamp" {
+		order[0] = "Time"
```

Comment:
Please add a comment for why do we need to do this.

Reply:
Same question, when does this condition happen?

Reply:
done.